src/app/app.component.ts

```diff
@@ -14,7 +14,7 @@ export class AppComponent implements OnInit {
 
     // Kick off services
     constructor(private appService: AppService,
-        private userService: UserService) {
+                private userService: UserService) {
```

Comment:
This spacing looks off to me, should it be back to the left 4 spaces?

Reply:
it came up in my lint options, told me to move it so it was aligned. i'm ambivalent, but we should make sure we have the same lint config